packages/terra-core-docs/src/terra-dev-site/doc/alert/AccessibilityGuide.2.doc.mdx

```diff
@@ -34,46 +37,100 @@ The user only expects to be interrupted for notifications (alerts) that are of h
 * The `"status"` role will not interrupt the user and present politely.
 
-<Notice variant="important" ariaLevel="5">
+#### Live regions
 
-###### Accessibility Guidance: Live regions
+- For less critical notifications (non-alerts), the notification banner should be in an `aria-live="polite"` region.
+    - For screen readers to pick up and announce a newly rendered notification, the notification banner must be within an aria-live region.
+    To enable assistive technologies the ability to inform the user of a less critical notification in a polite (non-interruptive) way, ensure the notification banner is
+    within an element where aria-live="polite".
+- Avoid mixing alert notifications and less critical notifications in the same region.
+    - The `"alert"` notification type/role implicitly has aria-live value of `assertive` and will interrupt without being in a region where aria-live was previously set.
+    It is recommended to separate critical alert notifications from less critical notification banners to prevent any confusion.
 
-For screen readers to pick up and announce a newly rendered notification, the notification must be within an aria-live region.
-To enable assistive technologies the ability to inform the user of a less critical notification in a polite (non-interruptive) way, ensure the Alert is
-within an element where `aria-live="polite"`.
+<div aria-label="Example code">
 
-Please note: the `"alert"` role implicitly has aria-live value of `assertive` and will interrupt without being in a region where aria-live is previously present.
+    import Alert from 'terra-alert';
+
+    <>
+        // This less critical notification should be in polite aria-live region so screen reader users are notified.
+        <div aria-live="polite">
+            {isOpen && (
+                <Alert type="success">
+                    This notification is a less critical and should be polite.
+                </Alert>
+            )}
+        </div>
+
+        // Alert notifications are implicitly assertive, no aria-live needed
+        // It is recommended to keep these separate from polite aria-live regions.
+        <div>
+            {isOpen && (
+                <Alert type="alert">
+                    This notification is a critical alert that is interruptive.
+                </Alert>
+            )}
+        </div>
+    </>
+
+</div>
+
+#### Custom Titles
+
+- Custom notification banners should have a title provided to the `title` prop in order to be accessible.
+
+##### Bad: Avoid notifications with no titles
+<CustomExampleNoTitle />
+<br />
+
+##### Good: Titles help users understand notifications
+<CustomExample />
 
 <div aria-label="Example code">
 
     import Alert from 'terra-alert';
 
     <div aria-live="polite">
         {isOpen && (
-            <Alert>
+            <Alert type="custom" title="Meaningful Notification Title">
```

Comment:
Should this title be "Help!" to match the example?

Reply:
I removed the example code so instead the consumer can use the `<Code>` button to show/hide the code

![image](https://github.com/cerner/terra-core/assets/37750902/4fd72f43-f53e-4412-bfc1-e797a7233611)